### ISSUE TITLE
advfort-quickfix current known issues for df adv beta

### ISSUE THREAD

**Issue by GGumby** [DFHack/dfhack#4530]
from [Here](https://raw.githubusercontent.com/warmist/scripts/advfort-quickfix/gui/advfort.lua)
Doing jobs above or below the player unit isnt possible.
Not all metalsmithforge reactions are available. you cannot make any weapons or armor.

**Comment by warmist** [comment]
Just note for myself: check if civ has those metal crafts

**Comment by JonathanLochridge** [comment]
> from [Here](https://raw.githubusercontent.com/warmist/scripts/advfort-quickfix/gui/advfort.lua) Doing jobs above or below the player unit isnt possible. Not all metalsmithforge reactions are available. you cannot make any weapons or armor.

By "above or below" do you mean stuff on different Z-levels?

**Comment by GGumby** [comment]
Yes. Say for example you carve a Dstair in the floor and you want to carve a UDstair beneath you, you will find you cannot perform this action. Of course, if the default keybindings were changed i have no knowledge of them but i did try a fair few different key combos including the default to no avail.

**Comment by JonathanLochridge** [comment]
That is something to look into, I can try to confirm if the bug also happens on linux, or I can reproduce it on another computer more generally.

I can look through the controls listed in the file itself to see if I can find something too there.
